demo/src/main/java/com/novoda/storagepathfinder/demo/LandingActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.novoda.storagepathfinder.demo
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.LinearLayoutManager
+import com.novoda.storagepathfinder.StoragePath
+import kotlinx.android.synthetic.main.activity_landing.*
+import java.util.*
+
+class LandingActivity : AppCompatActivity() {
+
+    private var assetCloner: AssetCloner? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_landing)
+
+        assetCloner = DemoDependenciesFactory.createAssetCloner(applicationContext)
+        val storageInspector = DemoDependenciesFactory.createStorageInspector(applicationContext)
+
+        val deviceStoragePaths = ArrayList<StoragePath>()
+        deviceStoragePaths.add(storageInspector.primaryStorageBasePath)
+        deviceStoragePaths.add(storageInspector.primaryStorageApplicationPath)
+        deviceStoragePaths.addAll(storageInspector.secondaryStorageBasePath)
+        deviceStoragePaths.addAll(storageInspector.secondaryStorageApplicationPath)
+
+        device_storage_roots.layoutManager = LinearLayoutManager(applicationContext)
```

Comment:
It's so nice to remove the `findViewById` ❤️

Reply:
With kotlin extensions I tend to either use camel case on the xml or use import alias:
```
import kotlinx.android.synthetic.main.activity_landing.device_storage_roots as deviceStorageRoots
```
But import alias is a bit confusing for some and since xml is already ugly I personally don't mind naming them with camel case there for the sake of the code looking noize :)